app-libs/exchange-oracle/src/coingecko.rs

```diff
@@ -33,7 +39,15 @@ const COINGECKO_PARAM_CURRENCY: &str = "vs_currency";
 const COINGECKO_PARAM_COIN: &str = "ids";
 const COINGECKO_PATH: &str = "api/v3/coins/markets";
 const COINGECKO_TIMEOUT: Duration = Duration::from_secs(3u64);
-
+//TODO from API ?
```

Comment:
I don't get this TODO - what do you mean with that?

Reply:
Check if we want to get coingecko coins' id from coingecko API ?